doc/Requirements Specification/Parts/Models.lyx

```diff
@@ -426,13 +809,31 @@ status open
 
 
 \backslash
-gls{PCM}
+glslabel{system}{software system}
+\end_layout
+
+\end_inset
+
+ will hold the predicted 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{QoS}
 \end_layout
 
 \end_inset
 
 .
- The little changes in the 
+ When analysing the imported model, they recognise significant differences
+ between the initial predictions for the prototype and the ones made for
+ the actual implementation.
+ The necessity to check copy right violations and content integrity as well
```

Comment:
[copyright](http://dict.leo.org/ende/index_de.html#/search=copyright&searchLoc=0&resultOrder=basic&multiwordShowSingle=on) is one word.